eos/statistics/log-likelihood.cc

```diff
@@ -1021,7 +1024,10 @@ namespace eos
                 gsl_matrix * covariance = gsl_matrix_alloc(k, k);
                 gsl_matrix_memcpy(covariance, _covariance);
 
-                return LogLikelihoodBlockPtr(new MultivariateGaussianBlock(cache, std::move(ids), mean, covariance, _number_of_observations));
+                gsl_matrix * response = gsl_matrix_alloc(k, k);
```

Comment:
see comment L818

Reply:
This is a bug. Thanks, fixed locally. Will push in a bit.